xformers/components/nvfuser/__init__.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+from pathlib import Path
+from typing import Any, Callable, Dict, Set, Union
+
+from xformers import _is_functorch_available
+from xformers.utils import (
+    generate_matching_config,
+    get_registry_decorator,
+    import_all_modules,
+)
+
+from .base import Fused, FusedConfig  # noqa
+
+# CREDITS: Classy Vision registry mechanism
+
+FUSED_REGISTRY: Dict[str, Any] = {}
```

Comment:
Perhaps rename this to NVFUSED_REGISTRY? TO distinguish from triton fused for example. But not sure we need this?